tests/functional/karate_tests/.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea/
```

Comment:
Need not required as parent directory already has the ignore file

Reply:
+1 this file shouldn't be needed.